README.md

```diff
@@ -1,169 +1,23 @@
-# Golang Bootcamp
```

Comment:
I don't see some useful information about what is your project about. Please add information regarding the project, what it does, which endpoints it has, and what information I could get from each one of these endpoints. Even some samples could help here.